src/Achievements/AchievementCategory.tsx

```diff
@@ -0,0 +1,46 @@
+import React from "react";
+
+import { Accordion, AccordionSummary, AccordionDetails, Typography } from "@mui/material";
+
+import { Achievement } from "./Achievements";
+
+interface IProps {
+  title: string;
+  achievements: { achievement: Achievement }[];
+  allAchievements?: { achievement: Achievement }[];
+  sx?: boolean;
```

Comment:
The `sx` prop name doesn't make sense, and also it isn't the right type for a usual `sx` prop.

I would either call this something like "pad", or just pass a full `sx` through.